ehitchhiking-ui/src/app/shared/components/trip/trip.module.ts

```diff
@@ -2,9 +2,10 @@ import {NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
 import {MatButtonModule, MatDialogModule, MatTableModule} from '@angular/material';
 import {TripComponent} from './trip.component';
+import { RatePassengersModalModule } from "../rate-passengers-modal/rate-passengers-modal.module";
 @NgModule({
 	declarations: [TripComponent],
-	imports: [CommonModule, MatButtonModule, MatDialogModule, MatTableModule],
+  imports: [CommonModule, MatButtonModule, MatDialogModule, MatTableModule, RatePassengersModalModule],
```

Comment:
pls move RatingComponent to separate module and inject a new module to RatePassengersModalModule, TripModule